pepper-apis/src/main/java/org/broadinstitute/ddp/util/TestRedisConnection.java

```diff
@@ -0,0 +1,37 @@
+package org.broadinstitute.ddp.util;
+
+import org.apache.commons.lang.StringUtils;
+import org.broadinstitute.ddp.constants.ConfigFile;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import redis.clients.jedis.Jedis;
+
+public class TestRedisConnection {
+    private static final Logger LOG = LoggerFactory.getLogger(TestRedisConnection.class);
+
+    public static void doTest() {
+        String redisAddress = ConfigManager.getInstance().getConfig().getString(ConfigFile.REDIS_SERVER_ADDRESS);
+        String host = StringUtils.substringBetween(redisAddress, "//", ":");
+        String portString = StringUtils.substringAfterLast(redisAddress, ":");
+        doTest(host, Integer.parseInt(portString));
+    }
+
+    public static void doTest(String host, int port) {
+        String testHash = "testhash";
+        String valToWrite = Math.random() + "";
+        try (var jedis = new Jedis(host, port)) {
```

Comment:
Using Jedis instead or Redisson. Jedis does not have a cache implementation. (Though Spring caching uses Jedis in their Redis cache implementation).